graphrag/index/cli.py

```diff
@@ -318,3 +322,56 @@ def _enable_logging(root_dir: str, run_id: str, verbose: bool) -> None:
         datefmt="%H:%M:%S",
         level=logging.DEBUG if verbose else logging.INFO,
     )
+
+# Function to validate config file for model/deployment name typos
+def _validate_config_names(
+    root: str,
+    config: str | None,
+    reporter: ProgressReporter) -> None:
+        import os
+        from graphrag.config.enums import LLMType
+        from graphrag.index.llm import load_llm, load_llm_embeddings
+        from datashaper import NoopVerbCallbacks
+        from graphrag.llm.types import (
+            LLM,
+            CompletionInput,
+            CompletionLLM,
+            CompletionOutput,
+            LLMInput,
+            LLMOutput,
+        )
+        
+        # Fetch raw list of params from config file
+        parameters = _read_config_parameters(root, config, reporter)
+        
+        # Validate Chat LLM configs
+        llm = load_llm(
+                "test-llm",
+                parameters.llm.type,
+                NoopVerbCallbacks(),
+                None,
+                parameters.llm.model_dump()
+            )
+        try:
+            asyncio.run(llm("This is an LLM connectivity test. Say Hello World"))
+            reporter.success("LLM Config Params Validated")
+        except Exception as e:
+            reporter.error("LLM Config Error Detected. Exiting...")
+            print(e)
```

Comment:
Can you remove the `print(e)` statement and instead add that to the `reporter.error(...)` message above. The reporter is a logger behind the scenes so there's no need to do an explicit print statement here. The logger will do that.

I think this would work.
`reporter.error(f"LLM configuration error detected. Exiting...\n{e}")`